src/Symfony/Component/Routing/Tests/Generator/Dumper/PhpGeneratorDumperTest.php

```diff
@@ -118,7 +118,6 @@ public function testDumpForRouteWithDefaults()
     public function testDumpWithSchemeRequirement()
     {
         $this->routeCollection->add('Test1', new Route('/testing', array(), array(), array(), '', array('ftp', 'https')));
-        $this->routeCollection->add('Test2', new Route('/testing_bc', array(), array('_scheme' => 'https'))); // BC
```

Comment:
All of these tests are useless because we already tested that logic inside the `Route` class that _scheme/_methods is backwards compatible and will result in the same route representation.